main_test.py

```diff
@@ -0,0 +1,82 @@
+import pytest
```

Comment:
please add auth path to your test cases
did you try running these test-cases?

Reply:
yes I tried & these are working but I have to add authentication on this also as you said.